Traceback (most recent call last):
  File "/home/jplal/MCGT/zz-scripts/chapter02/plot_fig03_relative_errors.py", line 28, in <module>
    plt.scatter()*T[ primary],
    ^^^^^^^^^^^^^
TypeError: scatter() missing 2 required positional arguments: 'x' and 'y'
